src/lib/getNavigationEntry.ts

```diff
@@ -16,9 +16,7 @@
 
 export const getNavigationEntry = (): PerformanceNavigationTiming | void => {
   const navigationEntry =
-    self.performance &&
-    performance.getEntriesByType &&
-    performance.getEntriesByType('navigation')[0];
+    globalThis.performance?.getEntriesByType?.('navigation')[0];
```

Comment:
Yeah, it's is baseline widely available, but we'd previously fixed these bugs, so that's why I was keeping it in:
- https://github.com/GoogleChrome/web-vitals/issues/185
- https://github.com/GoogleChrome/web-vitals/issues/159

Reply:
I wonder if it’s worth testing this branch in Opera Mini to see if there’s going to be any other similar issues?

Reply:
So, I just tested this in Opera Mini (with extreme data saver mode on) and discovered some interesting things.

- Looks like `performance.now()` is supported, so I guess they must have added that since I last tested.
- ES6 is not supported, so if we're going to be publishing ES6 then we probably don't need to worry about other quirks of Opera Mini on extreme data saver mode
- web-vitals v3 runs without errors, but v4 doesn't work for some reason (not sure what).

When I don't turn on extreme data saver mode, all of the code runs just fine since Opera uses Chromium.

Given all of this, and our intention to support Baseline Widely Available, I think it's fine to start accessing `globalThis.performance` without conditional check.

Anyone have concerns?